src/index.js

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+// eslint-disable-next-line import/extensions
+import App from './App.js';
```

Comment:
Dear @IsmaelMastronardi 
When we run the project, it shows the following error. This is because your App component file name is currently `app.js` which should be capitalized `App.js`. It starts working fine when we fix this typo. 

<img width="1437" alt="Screenshot 2023-08-09 at 12 33 35 AM" src="https://github.com/IsmaelMastronardi/math-magician/assets/83048208/6487158b-519c-4600-ab81-467c41592ee2">